File: src/assets/usage-examples-code/windows/set_current_window_named/set_current_window_named-1-basic-usage.cs
using System;
using SplashKitSDK;

string window1Name = "Window 1";
string window2Name = "Window 2";

Window wind1 = SplashKit.OpenWindow(window1Name, 800, 600);
Window wind2 = SplashKit.OpenWindow(window2Name, 800, 600);

Console.WriteLine("Setting the current window to '" + window1Name + "'...");
SplashKit.SetCurrentWindow(window1Name);
Console.WriteLine("Current window is now '" + window1Name + "'.");

SplashKit.Delay(1000); // Delay to keep the window open for 1 second

Console.WriteLine("Setting the current window to '" + window2Name + "'...");
SplashKit.SetCurrentWindow(window2Name);
Console.WriteLine("Current window is now '" + window2Name + "'.");

SplashKit.Delay(3000); // Delay to keep the window open for 3 seconds

File: src/assets/usage-examples-code/windows/set_current_window_named/set_current_window_named-1-basic-usage.cpp
#include "splashkit.h"

int main()
{
    string window1_name = "Window 1";
    string window2_name = "Window 2";

    window wind1 = open_window(window1_name, 800, 600);
    window wind2 = open_window(window2_name, 800, 600);

    write_line("Setting the current window to '" + window1_name + "'...");
    set_current_window(window1_name);
    write_line("Current window is now '" + window1_name + "'.");

    delay(1000); // Delay to keep the window open for 1 second

    write_line("Setting the current window to '" + window2_name + "'...");
    set_current_window(window2_name);
    write_line("Current window is now '" + window2_name + "'.");

    delay(3000); // Delay to keep the window open for 3 seconds

    return 0;
}
